### ISSUE TITLE
As a user I want to see client logo on the dashboard

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#71]

**Comment by lcabrini** [comment]
Hm.. the word 'client' is ambiguous here. Is it our client or our client's client?

**Comment by BorderITS** [comment]
check if this is ok

**Comment by lcabrini** [comment]
Yes, that is better. I would probably rather say (since we are viewing this from the point of the user): as a staff I want to see my company logo on the dashboard.

However, I'm not really sure this should be a user story at all. See my reasoning in #70

**Comment by BorderITS** [comment]
is this ok?

**Comment by lcabrini** [comment]
Yes, that is much better. However, it is still probably not a user story. But you leave it for now. I'm quite tired and not able to think that well any more. Tomorrow I'll take a decision.

**Comment by BorderITS** [comment]
Ok then
